.circleci/config.yml

```diff
@@ -201,115 +219,115 @@ workflows:
           name: x220-hotp-maximized
           target: x220-hotp-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: x220-maximized
           target: x220-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: t420-hotp-maximized
           target: t420-hotp-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: t420-maximized
           target: t420-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: x230-flash
           target: x230-flash
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: t430-flash
           target: t430-flash
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: t430
           target: t430
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: x230
           target: x230
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: x230-hotp-verification
           target: x230-hotp-verification
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: x230-maximized
           target: x230-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: t430-hotp-maximized
           target: t430-hotp-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: x230-hotp-maximized
           target: x230-hotp-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: t430-maximized
           target: t430-maximized
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: qemu-coreboot-fbwhiptail
           target: qemu-coreboot-fbwhiptail
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: kgpe-d16_workstation-usb_keyboard
           target: kgpe-d16_workstation-usb_keyboard
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: kgpe-d16_server
           target: kgpe-d16_server
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: kgpe-d16_server-whiptail
           target: kgpe-d16_server-whiptail
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: librem_l1um
           target: librem_l1um
           requires:
-            - librem_mini
+            - talos-2
 
       - build:
           name: librem_mini_v2
           target: librem_mini_v2
           requires:
-            - librem_mini
+            - talos-2
```

Comment:
Not sure why requires librem_mini was changed here for talos-2 (not same arch, not same set of dependencies) where from CircleCI perspective, what is important is [the dependency of save_cache to happen only after talos-2 has been built](https://github.com/osresearch/heads/pull/1002/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R208-R210)?

Reply:
Otherwise, we have to wait for another 1:58 minutes to build talos-2 prior of building all other boards, as the mapping shows [here](https://app.circleci.com/pipelines/github/SergiiDmytruk/heads/30/workflows/0e3e4ba6-79ab-4b36-867a-fc02520cc2ae)

Reply:
Initially it wasn't like this, but then I noticed that other `build` jobs are run after *all* `build_and_persist` and changed it in case it's important (e.g., if it's not OK to save cache while other jobs are running). I'll undo it.